<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scratchpadWindow</class>
 <widget class="QWidget" name="scratchpadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_selectFile">
     <property name="text">
      <string>select file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lbl_fileName">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No file selected.&lt;/p&gt;&lt;p&gt;To save your notes, select a file first. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="txt_notes"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lbl_status">
     <property name="text">
      <string>Nothing saved so far.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
